### ISSUE TITLE
Ovenplayer support for H265

### ISSUE THREAD

**Issue by yas1th** [AirenSoft/OvenPlayer#434]
Does the ovenplayer support H265 as of now ?

We were trying out ovenplayer for playback and provided 2 different master m3u8 files.

One with H265 codec and the other with H264 codec. 

I was able to play the H265 master file using ffplay.

However, when I try to load the ovenplayer with 2 different master m3u8 files , one with H265 & the other with H264. 

Ovenplayer is falling back to H264 master m3u8 file.

So just thought of checking on this if the ovenplayer supports H265 at the moment.

If it supports, can you please share some working  demo example or reference as well

**Comment by SangwonOh** [comment]
@yas1th OvenPlayer's hls playback uses hlsjs. Currently, hlsjs's h265 playback seems to only support fmp4 containers. Please check the container of your h265 stream?

And if you're playing on Chrome, you'll need to have hardware acceleration enabled in your browser settings.

**Comment by yas1th** [comment]
@SangwonOh : Yes, currently hls js is supporting fmp4 container. we were using MPEG-TS container hence it wasn't working as expected.

**Comment by zcolor** [comment]
OME just release v18 which implements srt publishing. Will OvenPlayer be updated to support SRT as well? I'm assuming that would simpify the use of h265?

**Comment by getroot** [comment]
OvenPlayer cannot support SRT because web browsers cannot support SRT
